sfall/Modules/Explosions.cpp

```diff
@@ -30,6 +31,7 @@ namespace sfall
 {
 
 static bool lightingEnabled = false;
+static bool explosionsMetarule = false;
```

Comment:
@FakelsHub , can you give a better name for this flag, like `needResetExplosionsAfterAttack`?